introduction/site_survey.md

```diff
@@ -0,0 +1,404 @@
+   > Note: Outside of a new installation of the CSM software, there is currently no supported method to rotate (change) the platform CA.
+   > Please ensure validity periods are set accordingly for external CAs you use in this process. The ability to rotate CAs is anticipated as part of a future release.
+
+| (Optional) LDAP Settings | Value |
+| ---- | ----- |
+| Is a site LDAP server available? | |
+| First site LDAP server | |
+| First site LDAP server port | |
+| (Optional) Second site LDAP server | |
+| (Optional) Second site LDAP server port | |
+| (Optional) Third site LDAP server | |
+| (Optional) Third site LDAP server port | |
+| site LDAP `ldapSearchBase` | |
+| site LDAP `localRoleAssignments` | |
+
+> Note: Setting `forwardZones` is needed if the site LDAP server is specified via a hostname rather than an IP address. See [Prepare Site Init](../install/prepare_site_init.md)
+
+<a name="application_nodes"></a>
+
+## 10.  Application nodes
+
+Each application node can have specific information about it.  Besides the CAN,some application nodes have additional network connections to Ethernet or InfiniBand.
+
+The only predefined Application node SHCD prefix and `subrole` is for UAN (User Access Node).
+
+| Each Application Node | Value |
+| ---- | ----- |
+| BMC or iLO username | |
+| BMC or iLO password | |
+| SHCD prefix | |
+| `subrole` | |
+| Hostname alias or aliases | |
+| Is CAN enabled for this node? | |
+| CAN IP address | |
+| CAN default route/gateway | |
+| CAN netmask | |
+| Network interface connected to CAN | |
+| Is network interface (`net1`) enabled for this node? | |
+| `net1` `bootproto` | |
+| `net1` device | |
+| `net1` IP address | |
+| `net1` `startmode` | |
+| `net1` `ifroute` route or routes | |
+| `net1` `ifrule` rules | |
+| Is network interface (`net2`) enabled for this node? | |
+| `net2` `bootproto` | |
+| `net2` device | |
+| `net2` IP address | |
+| `net2` `startmode` | |
+| `net2` `ifroute` route or routes | |
+| `net2` `ifrule` rules | |
+
+Common settings for all User Access Nodes (UANs).  These could be set the same for all application nodes rather than being set only for UANs.
+
+| Common UAN settings | Value |
+| ---- | ----- |
+| UAN global route or routes | |
+| UAN external DNS `searchlist` | |
+| UAN first external DNS server | |
+| UAN second external DNS server | |
+| UAN third external DNS server | |
+| UAN external DNS options | |
+| UAN LDAP enabled for login? | |
+| UAN LDAP domain | |
+| UAN LDAP `search_base` | |
+| UAN LDAP server or servers | |
+| UAN LDAP `chpass_uri` | |
+| UAN AD groups | |
+| UAN PAM modules | |
+
+<a name="filesystems"></a>
+
+## 11.  Filesystems
+
+For clients of a filesystem, there is some common data needed to be able to mount it.
+The filesystem type (`fstype`) could be Lustre, SpectrumScale (GPFS), or NFS.
+
+| Each Filesystem | Value |
+| ---- | ----- |
+| Filesystem name | |
+| Source IP address | |
+| `fstype` | |
+| Mount point | |
+| Mount options | |
+!
```

Comment:
Why the exclamation point on this line? Is that a Markdown directive I'm unfamiliar with?